pkg/common/telemetry/config.go

```diff
@@ -0,0 +1,31 @@
+package telemetry
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+type MetricsConfig struct {
+	FileConfig  FileConfig
+	Logger      logrus.FieldLogger
+	ServiceName string
+	Sinks       []Sink
+}
+
+type FileConfig struct {
+	Prometheus *PrometheusConfig `hcl:"Prometheus"`
+
+	DogStatsd []DogStatsdConfig `hcl:"DogStatsd"`
+	Statsd    []StatsdConfig    `hcl:"Statsd"`
+}
+
+type DogStatsdConfig struct {
+	Address string `hcl:"address"`
```

Comment:
I did it this way because it is consistent with the way that plugins are configured, and I also thought that perhaps in the future there will be additional config that might vary by instance/endpoint e.g. blacklisting or whitelisting certain metrics